packages/modules/minio/tsconfig.build.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "./tsconfig.json",
+  "exclude": [
+    "build",
+    "jest.config.ts",
+    "src/**/*.test.ts"
```

Comment:
We should exclude the dummy file here too

Reply:
Good point, added.